src/LanguageServer.Common/Utilities/DotNetRuntimeInfo.cs

```diff
@@ -189,7 +185,7 @@ public static List<DotnetSdkInfo> ParseDotNetListSdksOutput(TextReader dotNetLis
             string currentLine;
             while ((currentLine = dotNetListSdksOutput.ReadLine()) != null)
             {
-                Match parseResult = SdkInfoParser.Match(currentLine);
+                Match parseResult = SdkInfoParser().Match(currentLine);
```

Comment:
I’m unfamiliar with this syntax - I just want to be sure that we are not creating a new I instance of the regex every time we test for a match…?

Reply:
Ok, the docs say it caches a singleton instance so no worries there.